Pipfile

```diff
@@ -8,7 +8,7 @@ grpcio = "~=1.33.2"
 grpcio-tools = "~=1.33.2"
 requests = "~=2.25.0"
 python-json-logger = "~=0.1.11"
-cryptography = "~=3.2.1"
+cryptography = "~=3.3.2"
```

Comment:
why not the latest version so far: `3.4.4` ?

Reply:
I know that you have work in progress that use this library and also there is a PR from Glaucimar that use it. I just didn't want to have a bigger change to avoid possible issues, If you and Glaucimar are ok with the latest version, then I'm happy to update it.

Reply:
we are all using `3.x`,  they should be compatible, so I'm ok with the latest.

Reply:
I tested my WIP with `3.4.4`, it works.

Reply:
I also tested my WIP with 3.4.4 and it worked too.

Reply:
Great, thanks to both. I'll update it to the latest version then.